docker/node.dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM node:22.5-slim AS builder
 
-ARG GENESIS_FILENAME=testnet.json
+ARG GENESIS_FILENAME=localnet.json
```

Comment:
1. Is my recollection correct that this effectively changes the default genesis that indexer node will use?
2. Am I understanding correctly that this means that we would be switching that default from testnet to localnet, and if so, why?

Is there a reason that the default SHOULD NOT be the current testnet? That seems like the most reasonable default to me because it does not require the user to be running localnet to start using the indexer. Do you see it differently?